README.md

```diff
@@ -2,3 +2,24 @@
 
 # Drift Frontend
 
+# how to run
+
+* check out repo
+* cd to repo
+* `npm install`
+* `npm run start`
+
+If you see `ℹ ｢wdm｣: Compiled successfully.`, you are in good shape.
+
+# how to run with local drift-backend
+
+make a js file somewhere that looks like this
+
+```
+module.exports.routes = {
+           '/r/insights/platform/drift' : { host: 'http://localhost:8080' },
+}
+```
```

Comment:
Let's put this into the repository since it should be consistent for all users of drift-frontend to use at some point?

Reply:
Thoughts here @beav  ?

Reply:
I put in a PR to store it in drift-backend at https://github.com/RedHatInsights/drift-backend/pull/20/files. Once that's merged, I'll update this PR to reference that file.